### PR TITLE
Fix bert eval

### PR DESCRIPTION
**Pull request by yzhang123**

### CODE REVIEW

File: examples/nlp/language_modeling/bert_pretraining.py [hunk 1]
```diff
@@ -458,10 +460,16 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
     optimization_params['num_epochs'] = args.num_epochs
 else:
     optimization_params['max_steps'] = args.num_iters
+
+call_backs = [train_callback, ckpt_callback, eval_callback]
+
+if 'data_preprocessed' in sys.argv:
```
Comment (VahidooX) [inline examples/nlp/language_modeling/bert_pretraining.py:466]: Why not using args here?

File: examples/nlp/language_modeling/bert_pretraining.py [hunk 2]
```diff
@@ -404,9 +404,11 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
 # callback which prints training loss and perplexity once in a while
 if not args.only_mlm_loss:
     log_tensors = [train_loss, mlm_loss, nsp_loss]
+    eval_log_tensors = [eval_loss, eval_mlm_loss, eval_nsp_loss]
```
Comment (VahidooX) [inline examples/nlp/language_modeling/bert_pretraining.py:407]: Do you use all these tensors in the eval callbacks?

File: examples/nlp/language_modeling/bert_pretraining.py [hunk 3]
```diff
@@ -404,9 +404,11 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
...
     print_msg = "Loss: {:.3f} MLM Loss: {:.3f} NSP Loss: {:.3f}"
 else:
     log_tensors = [train_loss]
+    eval_log_tensors = [eval_loss]
     print_msg = "Loss: {:.3f}"
 train_callback = nemo_core.SimpleLossLoggerCallback(
     tensors=log_tensors,
```
Comment (VahidooX) [inline examples/nlp/language_modeling/bert_pretraining.py:414]: If you want to log evaluation losses in the TB, you should add them here too.

File: examples/nlp/language_modeling/bert_pretraining.py [hunk 4]
```diff
@@ -404,9 +404,11 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
 # callback which prints training loss and perplexity once in a while
 if not args.only_mlm_loss:
     log_tensors = [train_loss, mlm_loss, nsp_loss]
```
Comment (VahidooX) [inline examples/nlp/language_modeling/bert_pretraining.py:406]: How about changing its name to train_log_tensors to be consistent?
They are named train_tensors and eval_tensors in the scripts I worked on.

File: examples/nlp/language_modeling/bert_pretraining.py [hunk 5]
```diff
@@ -423,8 +425,8 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
     step_freq=args.save_step_freq,
 )
 
-ckpt_eval = nemo.core.EvaluatorCallback(
-    eval_tensors=[eval_mlm_loss, eval_nsp_loss],
+eval_callback = nemo.core.EvaluatorCallback(
+    eval_tensors=eval_log_tensors,
```
Comment (VahidooX) [inline examples/nlp/language_modeling/bert_pretraining.py:429]: I could not comment on the next section. Why not calculate total_steps outside of the if condition to have just one  lr_policy_fn = get_lr_policy()?

File: examples/nlp/language_modeling/bert_pretraining.py (hunk 1 above)
Comment (yzhang123) [reply examples/nlp/language_modeling/bert_pretraining.py:466]: its not a regular arg

File: examples/nlp/language_modeling/bert_pretraining.py (hunk 2 above)
Comment (yzhang123) [reply examples/nlp/language_modeling/bert_pretraining.py:407]: yes

File: examples/nlp/language_modeling/bert_pretraining.py (hunk 3 above)
Comment (yzhang123) [reply examples/nlp/language_modeling/bert_pretraining.py]: ok

**Pull request merged**